agent/pagebroker/daemon.cpp

```
@@ -0,0 +1,147 @@
+  response.set_request_id("");
+  response.set_transaction_id("");
+  response.mutable_failure()->set_code(Failure::INVALID_REQUEST);
+  response.mutable_failure()->set_message("invalid request");
+  return response;
+}
+
+void
+HandleConnection(int connection, Broker& broker)
+{
+  uint32_t size = 0;
+  if (!ReadAll(connection, &size, sizeof(size)))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)